hyper/ssl_compat_appengine.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+"""
+hyper/ssl_compat_appengine
+~~~~~~~~~
+
+Provides the ssl module interface which hyper assumes, based on AppEngine ssl.
+
+This module complements some constants and classes which don't exist on
+AppEngine's SSL module, to be used by hyper smoothly.
+See: https://cloud.google.com/appengine/docs/python/sockets/ssl_support
+"""
+
+import ssl
+
+from ssl import PROTOCOL_TLSv1, PROTOCOL_SSLv23, PROTOCOL_SSLv3
+from ssl import CERT_NONE, CERT_OPTIONAL, CERT_REQUIRED
+from ssl import match_hostname
+
+OP_NO_COMPRESSION = 0
+
+
+class SSLContext:
```

Comment:
This is a minor thing, but can we explicitly inherit from `object` here? It's unnecessary with Python 3, but old-style classes are a nasty surprise when working in Python 2 code and so it'd be good if we could save ourselves the hassle.